packages/dashboards/src/lib/configurator/components/widgets/configurator-items/data-source-error/data-source-error.component.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { DataSourceErrorComponent } from "./data-source-error.component";
```

Comment:
I just noticed that the `DataSourceErrorComponent` has a small leak. The `outputsSubject` subscription also needs a `takeUntil(this.onDestroy$)`.

Reply:
Wait never mind. ngOnDestroy already calls the dataSourceClear subject.